README.md

```diff
@@ -8,87 +8,698 @@ This simple PHP API client binds to ServerPilot's RESTful [API](https://github.c
+  * [Remove an existing app](#remove-an-existing-app)
+ 
+**Databases**
+
+  * [List all databases](#list-all-databases)
+  * [Retrieve an existing database](#retrieve-an-existing-database)
+  * [Create a new database](#create-a-new-database)
+  * [Update an existing database](#update-an-existing-database)
+  * [Remove an existing database](#remove-an-existing-database)
+ 
+**SSL**
+
+  * [Add custom SSL to an app](#add-custom-ssl-to-an-app)
+  * [Remove custom SSL from an app](#remove-custom-ssl-from-an-app)
+  * [Enable AutoSSL for an app](#enable-autossl-for-an-app)
+
+### Connect
 
 With your API `key` and `id` from ServerPilot, set up the config values and pass them to the ServerPilot class. You may alternatively include a `'decode' => false` config value if you just want the raw JSON-encoded value returned.
 
 ```php
 	$config = array(
-		'id' => 'cid_YOURID',
-		'key' => 'YOURKEY');
+		'id' => 'YOURID',
+		'key' => 'YOURKEY'
+	);
 	$sp = new ServerPilot($config);
 ```
 From there, you can call any number of functions to manage your ServerPilot servers, apps, system users, databases, etc.
 
-###Get lists:
+### Catch errors
+If there's a  problem with any request a `ServerPilotException` is thrown.
+
+You can retrieve the error message with `getMessage()` and the actual HTTP code with `getCode()`.
+
+```php
+try {
+    $servers    = $sp->server_list();
+} catch(ServerPilotException $e) {
+    echo $e->getCode() . ': ' .$e->getMessage();
+}
+```
+
+### Actions
+Actions are a record of work done on ServerPilot resources. These can be things like the creation of an App, deploying SSL, deleting an old Database, etc.
+
+All methods that modify a resource will return an `actionid` which can be used to track the status of said action.
+
+**Possible values of an action status**
+
+|Status|Description                                   |
+|-------|---------------------------------------------|
+|`open`   |Action has not completed yet.              |
+|`success`|Action was completed successfully.         |
+|`error`  |Action has completed but there were errors.|
+
+
+```php
+$sp->action_info('ACTIONID');
+```
+
+```json
+{
+  "data":
+  {
+    "id": "g3kiiYzxPgAjbwcY",
+    "serverid": "4zGDDO2xg30yEeum",
+    "status": "success",
+    "datecreated": 1403138066
+  }
+}
+```
+
+### Servers
+
+#### List all servers
+
 ```php
 $servers    = $sp->server_list();
-$users      = $sp->sysuser_list();
-$apps       = $sp->app_list();
-$databases  = $sp->database_list();
 ```
 
-###Get info on a particular resource:
+```json
+{
+  "data": [
+  {
+    "id": "FqHWrrcUfRI18F0l",
+    "name": "www1",
+    "autoupdates": true,
+    "firewall": true,
+    "lastaddress": "1.2.3.4",
+    "lastconn": 1403130552,
+    "datecreated": 1403130551
+  }, {
+    "id": "4zGDDO2xg30yEeum",
+    "name": "vagrant",
+    "autoupdates": true,
+    "firewall": true,
+    "lastaddress": "1.2.3.4",
+    "lastconn": 1403130554,
+    "datecreated": 1403130553
+  }]
+}
```

Comment:
I'd prefer to use tabs in all of these JSON examples, just for the sake of readability.